src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,16 +4,21 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
     @Override
     public Order completeOrder(Long userId) {
         // TODO: add log message about method completeOrder was called
+        logger.info("Method completeOrder was called. UserId-> '{}'", userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
         // NOTE: In production ready code this order identifier should be generated by DB
         // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.info("Method completeOrder. Order completed successfully. OrderId-> '{}'", order.getOrderId());
```

Comment:
The log message should be informative. Include the method name and parameters in the log message. For example, logger.info("Order completed successfully in completeOrder method. OrderId={}", order.getOrderId());